modules/apps/change-tracking/change-tracking-spi/src/main/java/com/liferay/change/tracking/spi/history/CTCollectionHistoryProviderUtil.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.change.tracking.spi.history;
+
+import com.liferay.osgi.service.tracker.collections.map.ServiceTrackerMap;
+import com.liferay.osgi.service.tracker.collections.map.ServiceTrackerMapFactory;
+
+import org.osgi.framework.BundleContext;
+
+/**
+ * @author Brooke Dalton
+ */
+public class CTCollectionHistoryProviderUtil {
+
+	public static ServiceTrackerMap<Long, CTCollectionHistoryProvider<?>>
+		getCTCollectionHistoryProviderServiceTracker(
+			BundleContext bundleContext) {
```

Comment:
The ServiceTrackerMap should be defined internally and we provide an api to retrieve the CTCollectionHistoryProvider by className or classNameId